phpstan.neon

```diff
@@ -11,7 +11,7 @@ parameters:
         - src
 
     scanDirectories:
-        - vendor
+        - /usr/share/icinga-php/vendor
```

Comment:
This is obsolete. At least, it succeeds even without this definition. So please drop it.

Reply:
It does not, which is why I added it anologous to the phpstan.neon located in the icinga-notifications-web repository:

![image](https://github.com/Icinga/ipl-orm/assets/146938323/58ef83d5-74f3-40a5-b090-dc6414850656)

Reply:
It success without the `scanDirectories` definition. It's an artifact of another change that was unnoticed till now.

Reply:
Where can I find that change? Because as I just said, it does not generate the baseline when used with a relative path. That being said, it runs without any issues when completely removing the `scanDirectories` directive.
Why was it used in the first place?

Reply:
git blame